variables.tf

```diff
@@ -1,61 +1,35 @@
-variable "namespace" {
-  type = string
-  default = "kube-prometheus-stack"
-}
-
 variable "chart_version" {
   type = string
+  description = "Chart version Prometheus-stack"
 }
 
-variable "grafana_pv_size" {
+variable "namespace" {
   type = string
-  default = "10Gi"
+  description = "This is the namespace used to install kube-prometheus-stack, it will be created."
+  default = "kube-prometheus-stack"
 }
 
 variable "prometheus_pv_size" {
   type = string
+  description = "Prometheus PV size"
   default = "100Gi"
 }
 
-variable "ingress_host" {
-  type = string
-}
-
-variable "basic_auth_username" {
-  type = string
-  default = "admin"
-}
-
-variable "cluster_issuer_name" {
-  type = string
-  default = ""
-}
-
-variable "install_adapter" {
+variable "prometheus_install_adapter" {
   type = bool
+  description = "Enable Prometheus Adapter"
   default = false
 }
 
-variable "grafana_ingress_enabled" {
-  type = bool
-  default = true
-}
-
-variable "adapter_chart_version" {
+variable "prometheus_adapter_chart_version" {
   type = string
+  description = "Chart version Prometheus Adapter"
   default = ""
```

Comment:
I think that we should not define a default. Without a default the error will be raised for `not defined variable` and not when the helm provider will try to install the chart

Reply:
Maybe we can be more explicit about the behaviour of this variable: if this variable is left empty the Prometheus Adapter will not be installed.